GDJS/Runtime/events-tools/soundtools.ts

```diff
@@ -428,6 +428,28 @@ namespace gdjs {
         runtimeScene
           .getSoundManager()
           .unloadAudio(soundFile, /* isMusic= */ true);
+
+      export const FadeVolume = (
+        runtimeScene: gdjs.RuntimeScene,
+        channel: integer,
+        toVolume: float,
+        timeOfFade: float
+      ) => {
+        //Clamping volume
+        if (toVolume > 100) {
+          toVolume = 100;
```

Comment:
I recommend that you define a new variable that would be

```js
const targetVolume = Math.max(0, Math.min(100, toVolume))
```

But then again, I'm not sure that we should add a max to the volume.

Reply:
Can you add the same positive check on duration and use seconds instead of milliseconds?